recipes/distopia/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "distopia" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/distopia-{{ version }}.tar.gz
+  sha256: 9118b6c6bf1589be3f7d8be0e295bdf737a9e2a13c54e6fac28f9aea8e50bf51
+
+build:
+  skip: True  # [py<38]
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  missing_dso_whitelist: "*/libc.so.6"
```

This should not be needed. Can you add `{{ compiler('c') }}` to avoid this below?
```suggestion
  missing_dso_whitelist: "*/libc.so.6"
```